src/selectField/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { Ref } from 'react'
+import cc from 'classcat'
+import { ChevronIcon } from 'icon'
+import { color } from '_utils/branding'
+import style from 'selectField/style'
+
+export interface SelectFieldProps {
```

Comment:
extends CommonFormFields (from TextField... maybe it should be moved to _utils?)

And for select I would add `multiple`

Reply:
Done for the `CommonFieldsProps`
For the `multiple` attribute I do not add it for now cause it totally break the component display.
Don't find a way to manage the component height according to number of option.
And we don't need this multiple options for now.

Reply:
Maybe it's not what you need but there is a `size` attribute you can pass along with `multiple` to choose how many options are displayed when opening the select

Reply:
You right `size` attribute fix the height issue but an other scrollbar display issue need to be fixed before adding `multiple` attribute option.
<img width="471" alt="capture d ecran 2019-02-27 a 15 00 17" src="https://user-images.githubusercontent.com/4067977/53500186-2b7d6400-3aaa-11e9-83f4-1c5995ab1f51.png">
I'm creating issues (multiple option for SelectField + CommonFieldsProps refacto for TextField) to keep post about what we plan to improve.